workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/container/ViewStateFrame.kt

```diff
@@ -29,7 +31,15 @@ internal data class ViewStateFrame(
   companion object CREATOR : Creator<ViewStateFrame> {
     override fun createFromParcel(parcel: Parcel): ViewStateFrame {
       val key = parcel.readString()!!
-      val viewState = parcel.readSparseArray<Parcelable>(ViewStateFrame::class.java.classLoader)!!
+      val viewState = if (VERSION.SDK_INT >= VERSION_CODES.TIRAMISU) {
```

Comment:
I'm kind of amazed they didn't give us compat wrappers to do all this version checking

Reply:
Should we make our own? We already have a module of Androidx support methods.

Reply:
Could consider it for sure.

Reply:
Did not do this as I think this type of branching is pretty standard practice and I'm ok with it as it makes it clear what can happen down the road rather than hiding that behind some syntactic sugar.